assets/.yarnrc.yml

```diff
@@ -1,3 +1,5 @@
 enableTelemetry: 0
 
 nodeLinker: node-modules
+
+yarnPath: .yarn/releases/yarn-4.3.1-git.20240705.hash-35167b2.cjs
```

Comment:
It seems incorrect to me to commit the .yarn directory and these changes seems incorrect to me to commit the .yarn directory and these changes

Reply:
😭️😭️😭️